### ISSUE TITLE
add remove background models in PyTorch

### ISSUE THREAD

**Issue by wolfkingal2000** [chaiNNer-org/spandrel#150]
add remove background like this models 
https://huggingface.co/briaai/RMBG-1.4
https://github.com/danielgatis/rembg
https://github.com/SkyTNT/anime-segmentation 
make can use this models in software tanks

**Comment by RunDevelopment** [comment]
> https://github.com/danielgatis/rembg

We already support that. See ONNX Remove Background.

**Comment by joeyballentine** [comment]
I still think it would be worth adding pytorch versions of rembg and potentially others as well

**Comment by wolfkingal2000** [comment]
> > https://github.com/danielgatis/rembg
> 
> We already support that. See ONNX Remove Background.

Yes, i use onnx vr, but I dont work my pc. i want to use pytorch vr

**Comment by martirsadota** [comment]
To add, I know of a bunch more background removal models:

InSPyReNet, the previous SOTA in Papers With Code: https://github.com/plemeri/InSPyReNet
BiRefNet, the current SOTA: https://github.com/plemeri/InSPyReNet
and TRACER: https://github.com/Karel911/TRACER
